client/src/fluid/FluidAutocomplete.ml

```diff
@@ -328,27 +328,32 @@ let secretToACItem (s : SecretTypes.t) : fluidAutocompleteItem =
   let asDval = DStr (Util.obscureString s.secretValue) in
   FACVariable (s.secretName, Some asDval)
 
+
 let isInQuery (tl : toplevel) ti =
   let ast' = TL.getAST tl in
   match ast' with
-  | None -> false
+  | None ->
+      false
   | Some ast ->
-    FluidAST.ancestors (FluidToken.tid ti.token) ast
-    |> List.find ~f:(function FluidExpression.EFnCall (_, name, _, _) -> if name = "DB::query_v4" then true else false | _ -> false)
-    |> Option.is_some
+      FluidAST.ancestors (FluidToken.tid ti.token) ast
+      |> List.find ~f:(function
+             | FluidExpression.EFnCall (_, name, _, _) ->
+                 if name = "DB::query_v4" then true else false
```

Comment:
There are a number of different DB::query functions so this won't support them all. I looked for calls to `User_db.query` in https://github.com/darklang/dark/blob/main/backend/libbackend/libdb2.ml:

- DB::query_v4
- DB::queryWithKey_v3
- DB::queryOne_v3
- DB::queryOne_v4
- DB::queryOneWithKey_v3
- DB::queryCount

Perhaps there should be an attribute for whether it's a query compiler function too? Otherwise we can just use a list.

Reply:
I wasn't sure what you meant by attribute here so I went with a list for starters.  (And sorry about the stupid commit message, I did a git amend and didn't noticed it overwrote the message until I'd already pushed it :\ )

For the "attribute" were you thinking of something like adding another field to the function_metadata record, maybe something like is_query_function and setting that based on a list defined in Sql_compiler like we did for compilerSupportedFns?

Reply:
> For the "attribute" were you thinking of something like adding another field to the function_metadata record, maybe something like is_query_function and setting that based on a list defined in Sql_compiler like we did for compilerSupportedFns?

Yes, exactly like that! And then in the future we can move it into the function definitions in libdb2, etc.